### PR TITLE
add an action to add new issues and PRs to project

### PR DESCRIPTION
**Pull request by cjyetman**
will auto add new issues and PRs to the [Transition Monitor project](https://github.com/orgs/RMI-PACTA/projects/8/)

### CODE REVIEW

**Review by AlexAxthelm — APPROVED** [review]

**Comment by jdhoffa** [comment]
I guess this should eventually get the same treatment as https://github.com/RMI-PACTA/pacta.data.preparation/pull/248 and friends re: using a generalized PAT

**Comment by cjyetman** [comment]
Yup. Just need the org level PAT with appropriate permissions to be created.

**Comment by AlexAxthelm** [comment]
I'll work  on setting that up

File: .github/workflows/add-prs-and-issues-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/RMI-PACTA/projects/8
+          github-token: ${{ secrets.CJ_ADD_TO_PROJECT_PAT }}
```
Comment (AlexAxthelm) [inline .github/workflows/add-prs-and-issues-to-project.yml:21]:
```suggestion
          github-token: ${{ secrets.PAT_ADD_ISSUES_TO_PROJECT }}
```

I've created the @PACTA-bot user who created the PAT_ADD_ISSUES_TO_PROJECT PAT with repo and project scopes.

**Review by AlexAxthelm — CHANGES_REQUESTED** [review]

**Review by AlexAxthelm — APPROVED** [review]

**Pull request merged**